docs/api-gateway/10-02-connect-api-rule.md

```diff
@@ -117,3 +117,27 @@ If you reach your service and get `404 Not Found` in response, make sure that:
   ```
 
   >**TIP:** Name of the VirtualService consists of the name of the APIRule and a random suffix.
+
+Sometimes Oathkeeper Maester controller stops reconciling Rules on long living clusters. This can result in random `404 Not Found` responses, because Oathkeeper does not contain Rules reflecting the actual state of the cluster. A simple restart of the Pod resolves the issue, but you might want to verify if that is the issue you have encountered.
```

Comment:
```suggestion
Sometimes Oathkeeper Maester controller stops reconciling Rules on long-living clusters. This can result in random `404 Not Found` responses, because Oathkeeper does not contain Rules reflecting the actual state of the cluster. A simple restart of the Pod resolves the issue, but you might want to verify if that is the issue you have encountered.
```